### PR TITLE
New Organizations Chapter

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/uyuni-project/uyuni-docs/pull/64

Editing new chapter in Admin Guide, content from Reference Guide.

### CODE REVIEW

File: modules/administration/pages/organizations.adoc [hunk 1]
```diff
@@ -1,141 +1,88 @@
...
-* [path]``salt_content_staging_window:`` expresses the duration, in hours, of the time window for Salt clients to stage packages in advance of scheduled installations or upgrades.
 
-A value of *salt_content_staging_advance* equal to *salt_content_staging_window* results in the content staging window closing exactly when the installation/upgrade is scheduled to be executed.
-A larger value allows separating download time from the installation time.
+=== Configure Organizations
 
-These options are configured in [path]``/usr/share/rhn/config-defaults/rhn_java.conf``
- and by default assume the following values:
+Navigate to the [guimenu]``Configuration`` tab to manage the configuration of your organization.
+This includes the use of staged contents, setting up crash reporting, and the use of of SCAP files.
```
Comment (keichwa) [inline modules/administration/pages/organizations.adoc]:
```suggestion
This includes the use of staged contents, setting up crash reporting, and the use of SCAP files.
```

File: modules/administration/pages/organizations.adoc [hunk 2]
```diff
@@ -1,141 +1,88 @@
...
 
-* [path]``salt_content_staging_advance: 8 hours``
-* [path]``salt_content_staging_window: 8 hours``
+For more information about content staging, see xref:administration:content-staging.adoc[].
```
Comment (keichwa) [inline modules/administration/pages/organizations.adoc]: If there are two or more items, we'd back make this an itemized list.

File: modules/administration/pages/content-staging.adoc [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+Staging is used by clients to download packages in advance, before they are installed.
+This allows package installation to begin as soon as it is scheduled, which can reduce the amount of time required for a maintenance window.
+
+
+== Enable Content Staging
+
+You can manage content staging across your entire organization.
+In the {productname} {webui}, navigate to menu:Admin[Organizations] to see a list of available organizations.
+Click on the name of an organization, and check the [guimenu]``Enable Staging Contents`` box to allow clients in this organization to stage package data.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/administration/pages/content-staging.adoc]: you're right :)

File: modules/administration/pages/content-staging.adoc [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+At the scheduled time, the staged packages are installed.
+
+
+
+== Configure Content Staging
+
+There are two parameters used to configure content staging:
+
+* [parameter]``salt_content_staging_advance`` is the advance time for the content staging window to open, in hours.
+This is the number of hours before installation that package download can begin.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/administration/pages/content-staging.adoc]: This is an awkward sentence no matter which word you choose.

File: modules/administration/pages/content-staging.adoc [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+Default values:
+
+* [path]``salt_content_staging_advance: 8 hours``
+* [path]``salt_content_staging_window: 8 hours``
+
+
+[NOTE]
+====
+Content staging must be enabled in order for these parameters to work correctly.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/administration/pages/content-staging.adoc]: yep.

File: modules/administration/pages/organizations.adoc [hunk 6]
```diff
@@ -1,141 +1,88 @@
-[[ref.webui.admin.org]]
+[[organizations]]
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/administration/pages/organizations.adoc]: not in an ID

File: modules/administration/pages/organizations.adoc [hunk 7]
```diff
@@ -1,141 +1,88 @@
...
+.Procedure: Creating a New Organization
+. In the {productname} {webui}, navigate to menu:Admin[Organizations], and click btn:[Create Organization].
+. In the [guimenu]``Create Organization`` dialog, complete these fields:
+* In the [guimenu]``Organization Name`` field, type a name for your new organization.
+The name should be between 3 and 128 characters long.
+* In the [guimenu]``Desired Login`` field, type the login name you want to use for the organization's administrator.
+This must be a new administrator account, you will not be able to use an existing administrator account to log in to the new organization, including the one you are currently logged in with.
+* In the [guimenu]``Desired Password`` field, type a password for the new organization's administrator.
+Confirm the password by typing it again in the [guimenu]``Confirm Password`` field.
+Password strength is indicated by the coloured bar beneath the password fields.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/administration/pages/organizations.adoc]: far out, my Australian was showing yesterday, clearly!

File: modules/administration/pages/organizations.adoc [hunk 8]
```diff
@@ -1,141 +1,88 @@
...
+* In the [guimenu]``Email`` field, type an email for the new organization's administrator.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/administration/pages/organizations.adoc]: I thought "email" on its own was acceptable these days ;)

File: modules/administration/pages/organizations.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/administration/pages/organizations.adoc]: I put it into prose instead, it's a pretty complicated concept.

File: modules/administration/pages/content-staging.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/administration/pages/content-staging.adoc]: Slight re-write.

File: modules/administration/pages/organizations.adoc (hunk 8 above)
[the remark this answers is not in the document]
Comment (keichwa) [reply modules/administration/pages/organizations.adoc]: > I thought "email" on its own was acceptable these days ;)

You are probably right.  You have the teenager around you daily ;) I must listen more to my nephews.

File: modules/administration/pages/content-staging.adoc [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+For example, if [parameter]``salt_content_staging_advance`` is set to six hours, and [parameter]``salt_content_staging_window`` is set to two hours, the staging window will open six hours before the installation time, and remain open for two hours.
+No packages will be downloaded in the four remaining hours until installation starts.
+
+If you set the same value for both [parameter]``salt_content_staging_advance`` and [parameter]``salt_content_staging_window`` packages will be able to be downloaded until installation begins.
+
+Configure the content staging parameters in [path]``/usr/share/rhn/config-defaults/rhn_java.conf``.
+
+Default values:
+
+* [path]``salt_content_staging_advance: 8 hours``
```
Comment (keichwa) [inline modules/administration/pages/content-staging.adoc]:
```suggestion
* [literal]``salt_content_staging_advance: 8 hours``
```

File: modules/administration/pages/content-staging.adoc [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+* [path]``salt_content_staging_window: 8 hours``
```
Comment (keichwa) [inline modules/administration/pages/content-staging.adoc]:
```suggestion
* [literal]``salt_content_staging_window: 8 hours``
```
Or just one code block? With
```
----
foo bar
----
````

File: modules/administration/pages/organizations.adoc [hunk 11]
```diff
@@ -1,141 +1,88 @@
...
 
-[[s2-sattools-org-details-conf]]
-== Organization Details > Configuration
+.Procedure: Creating a New Organization
+. In the {productname} {webui}, navigate to menu:Admin[Organizations], and click btn:[Create Organization].
+. In the [guimenu]``Create Organization`` dialog, complete these fields:
+* In the [guimenu]``Organization Name`` field, type a name for your new organization.
+The name should be between 3 and 128 characters long.
+* In the [guimenu]``Desired Login`` field, type the login name you want to use for the organization's administrator.
+This must be a new administrator account, you will not be able to use an existing administrator account to log in to the new organization, including the one you are currently logged in with.
```
Comment (keichwa) [inline modules/administration/pages/organizations.adoc]:
```suggestion
This must be a new administrator account, you will not be able to use an existing administrator account to sign in to the new organization, including the one you are currently signed in with.
```

File: modules/administration/pages/organizations.adoc [hunk 12]
```diff
@@ -1,141 +1,88 @@
...
-* menu:Organization Configuration[SCAP File Upload Size Limit]
-* menu:Organization Configuration[Allow Deletion of SCAP Results]
-* menu:Organization Configuration[Allow Deletion After (period in days)]
+From the menu:Admin[Organizations] section, you can access tabs to manage users, trusts, configuration, and states for your organization.
 
-When settings are done, confirm with clicking the btn:[Update Organization] button.
+[NOTE]
+====
+Organizations can only be managed by their administrators.
+To manage an organization, ensure you are logged in as the correct administrator for the organization you want to change.
```
Comment (keichwa) [inline modules/administration/pages/organizations.adoc]:
```suggestion
To manage an organization, ensure you are signed in as the correct administrator for the organization you want to change.
```

File: modules/administration/pages/organizations.adoc [hunk 13]
```diff
@@ -1,141 +1,88 @@
...
-The assigned contact method sets the time for when the next [command]``mgr_check`` will be executed.
 
-Next time an action is scheduled, packages will automatically be downloaded but not installed yet.
-When the scheduled time comes, the action will use the staged version.
+=== Trusted Organizations
 
-Minion Content Staging::
-Every Organization administrator can enable Content Staging from the Organization configuration page menu:Admin[Organization > OrgName > Configuration > Enable Staging Contents].
+Navigate to the [guimenu]``Trusts`` tab to add or remove trusted organizations.
+Establishing trust between organizations allow them to share content more easily between them, and gives you the ability to migrate clients from one organization to another.
```
Comment (keichwa) [inline modules/administration/pages/organizations.adoc]:
```suggestion
Establishing trust between organizations allow them to share content between them, and gives you the ability to migrate clients from one organization to another.
```

File: modules/administration/pages/content-staging.adoc (hunk 9 above)
Comment (Loquacity) [reply modules/administration/pages/content-staging.adoc]: i thought i changed these to [parameter] actually :confused:

**Pull request merged**